packages/vue-identicon/src/Identicon.ts

```diff
@@ -60,7 +60,7 @@ export const Identicon = Vue.extend({
     return {
       address: '',
       iconSize: DEFAULT_SIZE,
-      isAlternative: false,
+      isAlternativeIcon: false,
```

Comment:
You will need to make this change to the top-level `Data` interface on line 21 as well - this is why CI is failing.

Reply:
resolved in 78d54af

Reply:
resolved in 78d54af